go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/odpf/shield
 
 go 1.16
 
+replace github.com/jhump/protoreflect => /Users/kartik/go/src/github.com/jhump/protoreflect
```

Comment:
@krtkvrm wrong package

Reply:
oh... will remove this before merging
this is being used for debugging